crates/kitsune-captcha/src/hcaptcha.rs

```diff
@@ -0,0 +1,56 @@
+use crate::{error::CaptchaVerification, CaptchaBackend, ChallengeStatus, Result};
+use async_trait::async_trait;
+use http::Request;
+use serde::{Deserialize, Serialize};
+use typed_builder::TypedBuilder;
+
+use kitsune_http_client::Client;
+
+#[derive(Clone, TypedBuilder)]
+pub struct Captcha {
+    pub verify_url: String,
+    pub site_key: String,
+    pub secret_key: String,
+    #[builder(default = Client::builder().build())]
+    client: Client,
+}
+
+#[derive(Serialize, Deserialize, TypedBuilder)]
+struct Body {
+    secret: String,
+    response: String,
+    #[builder(default = None)]
```

Comment:
You don't have to explicitly declare a value here. When omitted, it will just call the `Default` trait impl

Same with the `client` field of the `Captcha` struct.

Reply:
Ah, that's good to know :D